src/components/SelectOption/SelectOption.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const SelectOption = ({ list }) => {
+const SelectOption = ({ list, setOptionValue, setMovieList, movieList }) => {
+  const sortMovies = e => {
+    e.target.value === 'reservationRate'
+      ? setOptionValue('reservationRate')
+      : setOptionValue('averageRate');
+  };
```

Comment:
```suggestion
  const sortMovies = e => {
      setOptionValue(e.target.value);
  };
```

아래 map 에서 list의 filter가 어떤 값인지 모르겠지만
만약 두가지 값만 있다면 이렇게 구성할 수 있겠네요!
만약 여러가지의 값인데, 단 두가지만 들어가면 되는 부분이라면 수정하지 않으셔도 됩니다!